app/views/app_groups/show.html.slim

```diff
@@ -90,6 +84,78 @@
               - if (@labels['app-group'].present?)
                 = @labels['app-group'][label]
 
+          hr.solid
+          - @app_group.helm_infrastructures.sort.each do |hi|
+            dd.col-sm-6.row
+              .col-sm-12
+                h4= hi.infrastructure_location.name.upcase
+              .col-sm-4 Capacity
+              .col-sm-8= "#{hi.helm_cluster_template.name.camelcase} (#{hi.max_tps} TPS)"
+              .col-sm-4 Provisioning Status:
+              .col-sm-8
+                = hi.provisioning_status
+
+              .col-sm-4 ArgoCD URL:
+              .col-sm-8
+                - url = hi.argo_application_url
+                a href=url Link
```

Comment:
let's show this only to barito super admin

Reply:
can we have target _blank, to open this on new tab?

Reply:
updated